Add $network, $remote_fs to Required-Start and Required-Stop. 
Add Short-Description.
Change default start and stop runlevel.
--- a/lldpad.init
+++ b/lldpad.init
@@ -37,12 +37,12 @@
 #
 ### BEGIN INIT INFO
 # Provides: lldpad
-# Required-Start: network
-# Required-Stop:
-# Default-Start: 3 5
-# Default-Stop:
-# Short Description: Link Layer Discovery Protocol Agent Daemon
-# Description: Link Layer Discovery Protocol Agent Daemon
+# Required-Start: $network $remote_fs
+# Required-Stop: $network $remote_fs
+# Default-Start: 2 3 4 5
+# Default-Stop: 0 1 6
+# Short-Description: Start and stop the lldp agent daemon
+# Description: Manage Link Layer Discovery Protocol Agent Daemon
 ### END INIT INFO
 
 LLDPAD=lldpad
